src/main/java/org/jabref/JabRefMain.java

```diff
@@ -186,4 +186,4 @@ private static void configureProxy(ProxyPreferences proxyPreferences) {
             Authenticator.setDefault(new ProxyAuthenticator());
         }
     }
-}
+}
```

Comment:
Please keep the newline at the end of the file. Just unstage this file from the commit.